python/chemiscope/sphinx/static/chemiscope-sphinx.js

```diff
@@ -12,12 +12,19 @@ const VISUALISER_MODE = {
 /**
  * Asynchronously loads the Chemiscope visualization for the dataset
  */
-async function loadChemiscopeSphinx(divId, filePath, visualizerMode = VISUALISER_MODE.DEFAULT) {
-    // Handle warnings
-    Chemiscope.addWarningHandler((message) => displayWarning(divId, message));
+async function loadChemiscopeSphinx(
+    divId,
+    filePath,
+    visualizerMode = VISUALISER_MODE.DEFAULT,
+    timeout = 4000
+) {
     // Display loading
     toggleLoadingVisible(divId, true);
 
+    const warnings = new Chemiscope.Warnings;
+    warnings.timeout = timeout; // defaults to 4s visibility
+    warnings.addHandler((message) => { displayWarning(divId, message, warnings.timeout)});
```

Comment:
I don't understand why `warnings.timeout` is needed here, since the timeout is more a property of the warning handler


```suggestion
    warnings.addHandler((message) => { displayWarning(divId, message, timeout)});
```

Reply:
I renamed this as `defaultTimeout` so it's clearer - one can always override from showMessage, but it's otherwise set there.